plugins/jobtype/src/azkaban/jobtype/HadoopPigJob.java

```diff
@@ -86,10 +85,14 @@ public HadoopPigJob(String jobid, Props sysProps, Props jobProps, Logger log)
     obtainTokens = getSysProps().getBoolean("obtain.binary.token", false);
     userPigJar = getJobProps().getBoolean("use.user.pig.jar", false);
 
+    // Allow user to enable fetching of HCatalog tokens
+    boolean obtainHcatToken = getJobProps().getBoolean(HadoopSecurityManager.OBTAIN_HCAT_TOKEN, false);
+    getSysProps().put(HadoopSecurityManager.OBTAIN_HCAT_TOKEN, Boolean.toString(obtainHcatToken));
```

Comment:
Good idea to allow admins to specify a default value for `obtain.hcat.token`. I've updated the pull request with this change.